consensus/polybft/state_store_epoch.go

```diff
@@ -129,9 +141,14 @@ func (s *EpochStore) getNearestOrEpochSnapshot(epoch uint64, dbTx *bolt.Tx) (*va
 }
 
 // insertEpoch inserts a new epoch to db with its meta data
-func (s *EpochStore) insertEpoch(epoch uint64, dbTx *bolt.Tx) error {
+func (s *EpochStore) insertEpoch(epoch uint64, dbTx *bolt.Tx, chainID uint64) error {
 	insertFn := func(tx *bolt.Tx) error {
-		epochBucket, err := tx.Bucket(epochsBucket).CreateBucketIfNotExists(common.EncodeUint64ToBytes(epoch))
+		chainIDBucket, err := tx.Bucket(epochsBucket).CreateBucketIfNotExists(common.EncodeUint64ToBytes(chainID))
+		if err != nil {
+			return err
+		}
+
+		epochBucket, err := chainIDBucket.CreateBucketIfNotExists(common.EncodeUint64ToBytes(epoch))
```

Comment:
```suggestion
		chainIDBucket, err := tx.Bucket(epochsBucket).CreateBucketIfNotExists(common.EncodeUint64ToBytes(chainID))
		if err != nil {
			return err
		}

		epochBucket, err := chainIDBucket.CreateBucketIfNotExists(common.EncodeUint64ToBytes(epoch))
```
Order is different than in initialize method where epoch bucket is created and then chainIDs subbuckets within epoch bucket. Here is vice versa.